dsgrid/cli/registry.py

```diff
@@ -686,6 +686,36 @@ def update_project(
         return 1
 
 
+@click.command()
+@click.pass_obj
+@click.pass_context
+@click.argument("project_id")
+@click.argument("filename", callback=_path_callback)
+@click.option(
+    "-l",
+    "--log-message",
+    required=True,
+    type=str,
+    help="Reason for addition",
```

Comment:
```suggestion
    help="Please specify the reason for this addition in your log message",
```

Reply:
Probably my suggested language doesn't fit with the rest of the CLI. How about, "Reason for adding these supplemental dimensions to the project"?